CONTRIBUTING.md

```diff
@@ -53,20 +53,21 @@ Install and enable the intellij's
 [save actions plugin](https://github.com/dubreuia/intellij-plugin-save-actions) 
 to reformat/refactor code automatically:
 
-Please turn on all the options except 
-  * Remove unused suppress warning annotation
-
-It allows us to do following when saving a file:
-* Organize imports
-* Reformat code based on the code style
-* Rearrange code (reorder methods, fields, etc.)
-* Add final to local variable
-* Add final to field
-* Remove explicit generic type for diamond
-* Qualify field access with this
-* Remove final from private method
-* Remove unnecessary semicolon
-* Add missing @Override annotations
+Please turn on the following options only:
```

Comment:
Specify the minimum version of the plugin that has these options?

Reply:
updated, thanks